worker/jobs/move-contact/move-contact-job.processor.ts

```diff
@@ -0,0 +1,75 @@
+import { env } from 'process'; 
+import { spawn } from 'child_process'; 
+
+import { JobProcessor, JobResult } from '../../worker';
+import { MoveContactParams } from './move-contact-job.worker';
+import { decryptSessionToken } from '../../../shared/encryption';
+
+
+export class MoveContactJobProcessor implements JobProcessor<MoveContactParams> {
+
+  readonly MAX_TIMEOUT = 180_000; // 30 minutes timeout?
+
+  async process(jobData: MoveContactParams): Promise<JobResult> {
+    try {
+      const { contactId, parentId, instanceUrl, sessionToken } = jobData;
+      const { ENCRYPTION_KEY } = env;
+
+      if (!sessionToken) {
+        return { success: false, message: 'Missing session token' };
+      } else if (!ENCRYPTION_KEY) {
+        return { success: false, message: 'Missing ENCRYPTION_KEY' };
+      }
+
+      const decryptedSessionToken = decryptSessionToken(
+        sessionToken, ENCRYPTION_KEY
+      );
+
+      const chtProcess = spawn('cht', [
+        `--url=${instanceUrl}`,
+        `--session-token=${decryptedSessionToken}`,
+        `--force`,
```

Comment:
is cht-conf's `upload_docs` folder properly cleaned up between runs?  we need to make sure nothing in that persists

Reply:
tested this and it seems cht-conf deletes it at the start of each run with `--force` mode

Reply:
Yes it deletes  files from the previous execution, but the current will be staying there, I will try to delete right after the execution.

Reply:
if it is cleaned up, then imo we don't think we need to do anything...  can keep it simple and let cht-conf handle it
in extreme cases like a crash during upload-docs, having these files not cleaned up might aid a recovery... ?  probably not. but maybe.

Reply:
Sure!